i18n/ja/docusaurus-plugin-content-docs/current/getting-started.md

```diff
@@ -22,7 +22,7 @@ Momento Serverless Cache はAPI ベースのサーバーレスサービスです
 
 まずはじめに、無料の認証トークンを作成し、キャッシュを作成し、そのキャッシュ上で`set` や`get` コマンドを実行してみましょう。以下を起動します。
 
-[![CLI lab](images/cli_lab.png)](https://play.instruqt.com/embed/momento/tracks/sandbox-container-1challenge?token=em_14J4EucaTHzL0rXw&finish_btn_target=_top&finish_btn_text=Return+to+Docs&finish_btn_url=https%3A%2F%2Fdocs.momentohq.com%2Fgetting-started#try-our-cli-and-an-sdk-in-your-browser)
+[![CLI lab](images/cli_lab_jp.png)](https://play.instruqt.com/embed/momento/tracks/hands-on-with-momento-cli-japanese?token=em_bsKxhS3nsrbYBv8H)
```

Comment:
I think you will want to provide the three `finish_btn_*` query parameters in your URL to make the final button do what we want it to do.